src/styles/search-container.css

```diff
@@ -0,0 +1,15 @@
+.search-container {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-container_big {
+  width: 100%;
+  height: 70px;
```

Comment:
Don't fix container size (if there is no such a requirement). The fixed height can lead to content overflow or accidental scroll bars if the content exceeds the container's height. Consider using padding and allow the content to define the height.